clindmri/scripts/fsl_bedpostx.py

```diff
@@ -57,10 +57,28 @@
   corresponding to the respective b-value.
 - The *.bvec files contain a 3x1 vector for each gradient, indicating the
   gradient direction.
-- The diffuson dta: ith volume in the data corresponds to a measurement
+- The diffuson data: ith volume in the data corresponds to a measurement
   obtained after applying a diffusion-sensitising gradient with a b-value given
-  by th ith entry in *.bval and a gradient direction given by the ith vector in
-  *.bvec
+  by the ith entry in *.bval and a gradient direction given by the ith vector
+  in *.bvec
+
+**Output files**
+
+For every model parameter a distribution of values is estimated.
+- mean_*samples.nii.gz files: the mean of this distribution is saved for every
+  parameter.
+- merged_*samples.nii.gz files: for the parameters used for tractography
+  (e.g. orientations), samples from these distributions are saved.
```

Comment:
Il will be nice to add a documentation on the advanced option too and add new option in the script?

Fibers: number of fibres modelled per voxel 
Wheight: multiplicative factor for the prior on the additional modelled fibres. A smaller factor means more weighting for the additional fibres
Burnin: number of iterations before starting the sampling. These might be increased if the data are noisy, and the MCMC needs more iterations to converge
Multi-Shell Model : Use this option if the data contain more than one non-zero bvalue. This will use the model described in Jbabdi et al. MRM 2012, where the diffusion coefficient is modelled using a Gamma distribution. This option will output the posterior mean of a diffusivity variance parameter mean_d_stdsamples. Command line argument: --model=2. 
Noise Floor Model : Use this option if (part of) the data is operating close to the noise floor. This will fit one extra parameter f0 that captures the noise floor. This is also described in Jbabdi et al. MRM 2012. This option will output the posterior mean of the noise floor parameter mean_f0samples. Command line argument: --f0 --ardf0. 
Rician Noise : Use this option to replace the default Gaussian noise assumption with Rician noise. This will output the posterior mean of a noise precision (i.e. inverse of variance) parameter mean_tausamples. Command line argument: --rician.